src/file_analysis/File.h

```diff
@@ -328,6 +328,9 @@ class File {
     detail::AnalyzerSet analyzers;       /**< A set of attached file analyzers. */
     std::list<Analyzer*> done_analyzers; /**< Analyzers we're done with, remembered here until they
                                             can be safely deleted. */
+    uint64_t seen_bytes;
```

Comment:
These should probably should go a few lines higher to avoid memory holes?

Also, these could use comments, i think can just copy the ones from files/main.zeek